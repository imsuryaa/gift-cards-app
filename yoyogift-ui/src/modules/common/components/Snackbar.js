import React, { Component } from "react";
import Snackbar from "@material-ui/core/Snackbar";
import SnackbarContent from "@material-ui/core/SnackbarContent";

class MySnackBar extends Component {
  render() {
    const { message , color} = this.props
    return (
      <div>
        <Snackbar
          anchorOrigin={{
            vertical: "bottom",
            horizontal: "right"
          }}
          open={true}
        >
          <SnackbarContent
            message={message}
            style={{backgroundColor: color}}
          />
        </Snackbar>
      </div>
    );
  }
}

export default MySnackBar;
