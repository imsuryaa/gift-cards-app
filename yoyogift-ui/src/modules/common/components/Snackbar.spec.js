import React from "react";
import { mount } from "enzyme";

import Snackbar from "./Snackbar";

describe("Footer Test Suite", () => {
  it("should render expected HTML", () => {
    expect(mount(<Snackbar message={"SAS"} color={"red"}/>).html()).toMatchSnapshot();
  });
});