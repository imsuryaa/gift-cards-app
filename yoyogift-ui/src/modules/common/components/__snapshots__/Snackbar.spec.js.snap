// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test Suite should render expected HTML 1`] = `"<div><div class=\\"MuiSnackbar-root-1 MuiSnackbar-anchorOriginBottomRight-5\\"><div class=\\"MuiTypography-root-38 MuiTypography-body1-47 MuiPaper-root-11 MuiPaper-elevation6-19 MuiSnackbarContent-root-8\\" role=\\"alertdialog\\" style=\\"background-color: red; webkit-transform: translate(0, 0); transform: translate(0, 0); webkit-transition: -webkit-transform 225ms cubic-bezier(0.0, 0, 0.2, 1) 0ms; transition: transform 225ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;\\"><div class=\\"MuiSnackbarContent-message-9\\">SAS</div></div></div></div>"`;
